cmd/fs-v1.go

```diff
@@ -113,12 +113,48 @@ func fsReaddirMetaBuckets(fsPath string) ([]string, error) {
 	return f.Readdirnames(-1)
 }
 
+// List of all bucket metadata configs.
 var bucketMetadataConfigs = []string{
 	bucketNotificationConfig,
 	bucketListenerConfig,
 	bucketPolicyConfig,
 }
 
+func migrateBucketMetadataConfigs(metaBucket, bucket, tmpBucket string) error {
+	for _, bucketMetaFile := range bucketMetadataConfigs {
+		_, err := fsStatDir(pathJoin(metaBucket, tmpBucket, bucketMetaFile))
```

Comment:
The code will become simpler/readable if you use os.Stat() and use the conditions fi.IsDir() and golang errors

It is better not to use fs-v1-helpers.go abstraction in the migration so that we have finer control